src/shared/modules/dbMeta/dbMetaDuck.ts

```diff
@@ -605,6 +633,13 @@ export const serverConfigEpic = (some$: any, store: any) =>
               }
               store.dispatch(setRetainCredentials(retainCredentials))
               value = retainCredentials
+            } else if (name === 'browser.retain_editor_history') {
+              let retainHistory = true
```

Comment:
here we seem to default to true? But not in the intial state?

Reply:
I copied the logic from `retain_connection_credentials` and it doesn't default to true per se. It only stays true if the config exists and is not falsy...